clients/python/tests/test_client.py

```diff
@@ -77,6 +77,35 @@ def test_register_existing_version(client: ModelRegistry):
         client.register_model(**params)
 
 
+@pytest.mark.e2e
+async def test_update_models(client: ModelRegistry):
```

Comment:
I would like to also have under e2e another scenario.

- register_model, as now
- instantiate a brand new ModelArtifact (or other entity) manually, not from a "get call"
- set the ModelArtifact `.id` (any other required property) and a new `.description`
- show the update call correctly updated the description, but did not nullify the model_format_name, model_format_version, etc.

I think this will be important to "document" behaviour for user who may expect the logical model entities to be akin to data values. wdyt ?

Reply:
updated wdyt?